src/helpers/validateSignUpEmailAddress.js

```diff
@@ -0,0 +1,13 @@
+import { ALPHANUMERIC } from "./validateEmailAddress";
+
+const validateSignUpEmailAddress = (email) => {
+    const emailDomain = process.env.NEXT_PUBLIC_SIGN_UP_EMAIL_DOMAIN
```

Comment:
Can this just be `SIGN_UP_EMAIL_DOMAINS`, please, `NEXT_PUBLIC` isn't necessary.

Reply:
To access the environment variables from the browser, I had to prefix the env variables with NEXT_PUBLIC. Else it will be undefined. 
Let me know if there is a better way of doing this? TIA

Reply:
Aha, I didn't know this. :)
That's fine in that case.

Reply:
Yeah, it's quite cool. 🙃